dagmc_stats/DagmcStats.py

```diff
@@ -7,7 +7,7 @@
 class DagmcStats:
     entity_types = [types.MBVERTEX, types.MBTRI, types.MBENTITYSET]
 
-    def __init__(self, filename, populate=False):
+    def __init__(self, filename, populate=False, entity=entity_types):
```

Comment:
I don't foresee ever calling the constructor with a different set of entity types? Is there a particular use case imagined here?

Reply:
I think this is the way to access the class variable in __init__() function, otherwise it will cause error.